### PR TITLE
Remove TestWorkspace constructors that are given a list of lines.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
THese helpers were used to make VB workspace construction easier when VB didn't have
an easy way to make multi-line strings.

Now htat VB can make multi-line strings, we no longer need these entrypoints.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-ide  Plz review.

**Comment by brettfo** [comment]
That's the most painful diff I've seen in a long time, but looks good.  :shipit:

**Comment by DustinCampbell** [comment]
Oh thank you! This is something that was on my list to go do at some point. :+1:

File: src/EditorFeatures/Test/Workspaces/TestWorkspaceFactory.cs [hunk 1]
```diff
@@ -57,33 +57,31 @@ public partial class TestWorkspaceFactory
...
+            string content)
         {
-            var total = content.Join(Environment.NewLine);
-            return CreateWorkspaceFromFilesAsync(workspaceKind, language, compilationOptions, parseOptions, total);
+            return CreateWorkspaceFromFilesAsync(workspaceKind, language, compilationOptions, parseOptions, new[] { content });
         }
 
-        /// <param name="files">Can pass in multiple file contents: files will be named test1.cs, test2.cs, etc.</param>
-        internal static Task<TestWorkspace> CreateWorkspaceFromFilesAsync(
+/// <param name="files">Can pass in multiple file contents: files will be named test1.cs, test2.cs, etc.</param>
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Test/Workspaces/TestWorkspaceFactory.cs]: I don't think you meant to do this.

**Comment by jasonmalinowski** [comment]
:+1: other than the one screwy formatting case. If you know which commit broke that test you had to fix, squashing with that commit might be nice.

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Pull request merged**